js/scripts.js

```diff
@@ -0,0 +1,15 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'triceratops';
+
+dinosaur = dinosaur.toUpperCase();
+
+var dinosaurCharsAfter = text.replace('Velociraptor', dinosaur);
+
+var halfSentence = dinosaurCharsAfter.length/2;
+
+dinosaurCharsAfter = dinosaurCharsAfter.substring(0, halfSentence);
+
+console.log(dinosaurCharsAfter);
+
+
```

Comment:
remove unnecessary empty lines - leave only one at the end of document

Reply:
You want me to remove only the last lines in this file? Maybe you want me also to remove some empty lines between the code? (lines 2-12). I'll try to guess by my own and send you the file.